tfhe/docs/README.md

```diff
@@ -43,7 +43,7 @@ Take a deep dive into TFHE-rs, exploring APIs from the highest to the lowest lev
 * [Core crypto API](references/core-crypto-api/): Low-level API with the primitive functions and types of the TFHE scheme
 * [TFHE deep dive](explanations/tfhe-deep-dive.md): Resources that explain the Fully Homomorphic Encryption scheme - TFHE
 
-### Support
+### Supports
```

Comment:
Just to confirm supports with an S, looking at e.g. Apple with lots of support categories, they have support without s

![image](https://github.com/zama-ai/tfhe-rs/assets/49040125/61ba1a37-6abc-4f01-b75a-b7b03c1cc3af)

Reply:
Support channels